Src/BootCamp.Chapter/Logger/FileLogger.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class FileLogger : ILogger
+    {
+        public void LogMessage(DateTime dateTime, string message)
```

Comment:
DateTime is not really needed.
It doesn't make sense to log any other time than DateTime.Now.
If you want date to sometimes be there, sometimes not, then just pass it use another method or configuration. I imagine that you either want dates always or never, never a mix.

Reply:
Removed this parameter and put the DateTime.Now in the message itself.